curl-helper.c

```diff
@@ -3744,6 +3748,11 @@ CURLOptionMapping implementedOptionMap[] =
   HAVE(SSL_OPTIONS),
 #else
   HAVENOT(SSL_OPTIONS),
+#endif
+#if HAVE_DECL_CURLOPT_AWS_SIGV4
+  HAVE(AWS_SIGV4),
+#else
+  HAVENOT(AWS_SIGV4),
```

Comment:
better not change the allocated ctor numbers, ie all new options should be added to the end of the mapping